src/apps/dashboard/modal/ReservationsGroupModal.tsx

```diff
@@ -43,6 +50,26 @@ const ReservationGroupModal: FC<ReservationGroupModalProps> = ({
     string[]
   >([]);
 
+  const { isSuccess: isSuccessPublizon, data: publizonData } =
```

Comment:
Would it make sense to also destructure `isLoading` or an equivalent and introduce a loader/skeletons to the page?